public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -62,29 +62,10 @@ define(function(require) {
       this.updateDialog(publishUrl, true);
     }
 
-    // When there are file events triggered by the editor, update the publish/republish status
-    function handleFileEvent() {
-      if(publisher.needsUpdate) {
-        return;
-      }
-
-      if(!Project.getPublishUrl()) {
-        return;
-      }
-
-      publisher.handlers.unpublishedChangesPrompt();
-      Project.publishNeedsUpdate(true, function(err) {
-        if(err) {
-          console.error("[Thimble] Failed to set the publishNeedsUpdate flag after a file change: ", err);
-          return;
-        }
-        publisher.needsUpdate = true;
-      });
-    }
-    bramble.on("fileChange", handleFileEvent);
-    bramble.on("fileDelete", handleFileEvent);
-    bramble.on("fileRename", handleFileEvent);
-    bramble.on("folderRename", handleFileEvent);
+    bramble.on("fileChange", publisher.showUnpublishedChangesPrompt);
```

Comment:
I think I might not have been super clear here, I apologize for that! What I meant was this line (and subsequent ones) should be:
```js
bramble.on("fileChange", publisher.showUnpublishedChangesPrompt.bind(publisher));
bramble.on("fileDelete", publisher.showUnpublishedChangesPrompt.bind(publisher));
...
```